projects/scion/e2e-testing/src/helper/testing.util.ts

```diff
@@ -134,3 +134,15 @@ export function withoutUndefinedEntries<T>(object: Record<string, T>): Record<st
     return acc;
   }, {} as Record<string, T>);
 }
+
+/**
+ * Resolves to the identity of the perspective active in the given {@link Page}, or rejects if no perspective is active.
+ */
+export async function getPerspectiveId(page: Page): Promise<string> {
```

Comment:
Can we reuse `parsePerspectiveIdFromWindowName()` from `WorkbenchPerspectiveService`?

Reply:
No, because in the context of playwright runner (node app).